tools/llvm-kompile-compute-ordinal/main.cpp

```diff
@@ -0,0 +1,99 @@
+#include <kllvm/ast/AST.h>
+#include <kllvm/parser/KOREParser.h>
+
+#include <llvm/Support/CommandLine.h>
+
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+#include <string>
+
+using namespace llvm;
+using namespace kllvm;
+
+cl::OptionCategory ordinal_cat("llvm-kompile-compute-ordinal options");
+
+cl::opt<std::string> kompiled_dir(
+    cl::Positional, cl::desc("<kompiled-dir>"), cl::Required,
+    cl::cat(ordinal_cat));
+
+cl::opt<std::string> line(
+    cl::Positional, cl::desc("<line>"), cl::Required, cl::cat(ordinal_cat));
+
+cl::opt<bool> is_k_line(
+    cl::Positional, cl::desc("[is-k-line]"), cl::init(false),
+    cl::cat(ordinal_cat));
+
+int64_t get_location(kore_axiom_declaration *axiom) {
+  auto *location_att
+      = axiom->attributes().get(attribute_set::key::Location).get();
+  assert(location_att->get_arguments().size() == 1);
+
+  auto *str_pattern = dynamic_cast<kore_string_pattern *>(
+      location_att->get_arguments()[0].get());
+  std::string location = str_pattern->get_contents();
+
+  size_t l_paren = location.find_first_of('(');
+  size_t first_comma = location.find_first_of(',');
+  size_t length = first_comma - l_paren - 1;
+  return std::stoi(location.substr(l_paren + 1, length));
+}
+
+int64_t get_k_location(std::string &definition) {
```

Comment:
This function returns an ordinal (not a location) so it should be renamed appropriately.